src/board/devices/output/7seg.ts

```diff
@@ -1,14 +1,35 @@
 import { Device } from 'board/devices/device'
 import { Byte, Word } from '../../../types/binary'
-
+enum SEVENsegNr {
+  _0011_1111 = 0,
+  _0000_0110 = 1,
+  _0101_1011 = 2,
+  _0100_1111 = 3,
+  _0110_0110 = 4,
+  _0110_1101 = 5,
+  _0111_1101 = 6,
+  _0000_0111 = 7,
+  _0111_1111 = 8,
+  _0110_1111 = 9,
+  _0111_0111 = 10,//a
+  _0111_1100 = 11,//b
+  _0100_1100 = 12,//c
+  _0101_1110 = 10,//d
+  _0111_1001 = 10,//e
+  _0111_0011 = 10,//f
```

Comment:
please use unique values for enum probably 13 for d, 14 for e and 15 for f or so